sklearn/utils/_plotting.py

```diff
@@ -186,39 +185,36 @@ def _get_line_kwargs(
         summary_value_name : str or None
             Name of the summary value provided in `summary_values`.
 
-        fold_line_kwargs : dict or list of dict
+        line_kwargs : dict or list of dict
             Dictionary with keywords passed to the matplotlib's `plot` function
             to draw the individual ROC curves. If a list is provided, the
             parameters are applied to the ROC curves sequentially. If a single
             dictionary is provided, the same parameters are applied to all ROC
             curves. Ignored for single curve plots - pass as `**kwargs` for
             single curve plots.
 
-        default_line_kwargs : dict, default=None
-            Default line kwargs to be used in all curves, unless overridden by
-            `fold_line_kwargs`.
-
         **kwargs : dict
             For a single curve plots only, keyword arguments to be passed to
-            matplotlib's `plot`. Ignored for multi-curve plots - use `fold_line_kwargs`
+            matplotlib's `plot`. Ignored for multi-curve plots - use `line_kwargs`
             for multi-curve plots.
         """
         # Ensure parameters are of the correct length
         name_ = [None] * n_curves if name is None else name
         summary_value_ = [None] * n_curves if summary_value is None else summary_value
-        # `fold_line_kwargs` ignored for single curve plots
-        # `kwargs` ignored for multi-curve plots
-        if n_curves == 1:
-            fold_line_kwargs = [kwargs]
-        else:
-            # Should we add an extra check ensuring `fold_line_kwargs` is the
-            # same when `summary_value` is (mean, std) ? or allow this flexibility
-            # when using `plot` - note this is checked and prevented in
-            # `from_cv_results`
-            fold_line_kwargs = cls._validate_line_kwargs(n_curves, fold_line_kwargs)
 
-        if default_line_kwargs is None:
-            default_line_kwargs = {}
+        # Can't have both `line_kwargs` and `kwargs`
+        if line_kwargs and kwargs:
+            raise ValueError(
+                "Cannot provide both `line_kwargs` and `kwargs`. Pass all "
+                "matplotlib arguments to `line_kwargs`."
```

Comment:
Should we mention that `kwargs` is depreacted?

Reply:
done